js/gatecoin.js

```diff
@@ -447,19 +450,41 @@ module.exports = class gatecoin extends Exchange {
                 throw new AuthenticationError (this.id + ' two-factor authentication requires a missing ValidationCode parameter');
         }
         let response = await this.privatePostTradeOrders (this.extend (order, params));
-        return {
+        let result = {
             'info': response,
             'id': response['clOrderId'],
         };
+        if ('responseStatus' in response) {
```

Comment:
Can you please move these two repeating sections to `handleErrors`? (see liqui, binance and other exchanges for example).

Reply:
Those two sections are not error handling; I am trying to deduct if the request passed based on gatecoin vague response. I.e. I am handling a successful case, not an error case. 

I have just looked into `liqui.js` and it seems that the successful state is ignored in `handleErrors` and only failed states are parsed instead. 

Could you please elaborate on the course of action? I will do.